packages/babel-plugin-transform-classes/src/transformClass.ts

```diff
@@ -219,6 +218,20 @@ export default function transformClass(
       if (t.isClassMethod(node)) {
         const isConstructor = node.kind === "constructor";
 
+        if (!process.env.BABEL_8_BREAKING && !USE_ESM && !IS_STANDALONE) {
+          // polyfill when being run by an older Babel version
+          path.ensureFunctionName ??=
+            // eslint-disable-next-line no-restricted-globals
+            require("@babel/traverse").NodePath.prototype.ensureFunctionName;
+        }
+        path.ensureFunctionName(supportUnicodeId);
+        let wrapped;
+        if (node !== path.node) {
+          wrapped = path.node;
+          // The node has been wrapped. Reset it to the original once, but store the wrapper.
+          path.replaceWith(node);
+        }
+
```

Comment:
![image](https://github.com/user-attachments/assets/3505d4c9-665a-47ce-90c7-c463b714ccea)
Moving them here will also pass the test. Is this better?

Reply:
Yes